src/frontend/apps/lti_site/components/LocalizedTimedTextTrackUpload/LanguageSelect/index.spec.tsx

```diff
@@ -20,18 +19,26 @@ jest.mock('lib-components', () => ({
 const onChangeMock = jest.fn();
 
 const languageChoices = [
-  { label: 'English', value: 'en' },
-  { label: 'French', value: 'fr' },
-  { label: 'Spanish', value: 'es' },
-  { label: 'Slovenian', value: 'sl' },
-  { label: 'Swedish', value: 'sv' },
+  { display_name: 'English', value: 'en' },
+  { display_name: 'French', value: 'fr' },
+  { display_name: 'Spanish', value: 'es' },
+  { display_name: 'Slovenian', value: 'sl' },
+  { display_name: 'Swedish', value: 'sv' },
 ];
 
 describe('<LanguageSelect />', () => {
-  it('renders the component with instructor local language available', () => {
-    useTimedTextTrackLanguageChoices.setState({
-      choices: languageChoices,
-    });
+  afterEach(() => {
```

Comment:
I get lot of errors when I launch the test, considere to mock console.error or mock the missing translation.

console.error
    Error: [@formatjs/intl Error MISSING_TRANSLATION] Missing message: "components.LanguageSelect.selectLanguageLabel" for locale "fr-FR", using default message as fallback.

Reply:
The best would have been to do something like that:
```
jest.mock(
  'translations/fr-FR.json',
  () => ({
    'components.LanguageSelect.noLanguageAvailableLabel': 'Pas de language',
    'components.LanguageSelect.selectLanguageLabel': 'Selectionne language',
  }),
  { virtual: true },
);
```
But we cannot do it with the render implementation though..
Another clean way, would have been to change a bit `intl.tsx` in `lib-test` to be able to load internationalized messages, but it is a bit out of your scope.

So, I would recommend to mock console error.
```
jest.spyOn(console, 'error').mockImplementation(() => jest.fn());
```

Reply:
The best would have been to do something like that:
```
jest.mock(
  'translations/fr-FR.json',
  () => ({
    'components.LanguageSelect.noLanguageAvailableLabel': 'Pas de language',
    'components.LanguageSelect.selectLanguageLabel': 'Selectionne language',
  }),
  { virtual: true },
);
```
But we cannot do it with the render implementation though..
Another clean way, would have been to change a bit `intl.tsx` in `lib-test` to be able to load internationalized messages, but it is a bit out of your scope.

So, I would recommend to mock console error.
```
jest.spyOn(console, 'error').mockImplementation(() => jest.fn());
```

Reply:
e

Reply:
The best would have been to do something like that:
```
jest.mock(
  'translations/fr-FR.json',
  () => ({
    'components.LanguageSelect.noLanguageAvailableLabel': 'Pas de language',
    'components.LanguageSelect.selectLanguageLabel': 'Selectionne language',
  }),
  { virtual: true },
);
```
But we cannot do it with the render implementation though..
Another clean way, would have been to change a bit `intl.tsx` in `lib-test` to be able to load internationalized messages, but it is a bit out of your scope.

So, I would recommend to mock console error.
```
jest.spyOn(console, 'error').mockImplementation(() => jest.fn());
```

Reply:
What I did with Vassili was adding this in `intl.tsx` : 
```
  <IntlProvider locale={locale} onError={() => {}}>
    {Component}
  </IntlProvider>
```
But I'm not sur about committing it

Reply:
Ok I get I now, thank